YAMP.Core/Help/Documentation.cs

```diff
@@ -307,7 +307,9 @@ static String GetLocalized(String key)
         {
             var value = default(String);
             var dictionary = Localization.Current ?? Localization.Default;
-            dictionary.TryGetValue(key, out value);
+            //Fallback to show something while key/descriptions are being replaced
+            if (!dictionary.TryGetValue(key, out value))
+                value = key;
```

Comment:
I thought the same as you, but then I saw (if i remember well) the YAMP.Core.Tests.Examples.DefaultLocalizationIsWellConnected, which seems to test it explicitly, and you would use it to spot the missing keys.